superagi/tools/duck_duck_go/duck_duck_go_search.py

```diff
@@ -90,7 +90,7 @@ def get_formatted_webpages(self,search_results,webpages):
         for webpage in webpages:
             results.append({"title": search_results[i]["title"], "body": webpage, "links": search_results[i]["href"]})
             i += 1
-            if TokenCounter.count_text_tokens(json.dumps(results)) > 3000:
+            if TokenCounter().count_text_tokens(json.dumps(results)) > 3000:
```

Comment:
initate TokenCounter

Reply:
done

Reply:
done sir made the rest of the methods as static method